src/components/data-processing/ReadAlignment/PlotStyleMisc.jsx

```diff
@@ -1,28 +1,25 @@
-import React, { useRef } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import {
   Slider, Form,
 } from 'antd';
 
 const BandwidthOrBinstep = (props) => {
   const {
-    onUpdate, config, type, max,
+    onUpdate, config, type, max, min,
   } = props;
-  let min = 0.001;
-  let maxDefault = 0.2;
-  if (max) {
-    min = max / 4;
-    maxDefault = max;
-  }
-  const step = maxDefault / 200;
+
+  const realMin = min ?? max / 4;
```

Comment:
I refactored this a little bit becuase it was very confusing and much of this didn't seem to work.

Max will never be null or undefined (becuase of defaultProps), so min will always be max/4